desktop/src/lib/video-utils.ts

```diff
@@ -24,87 +24,88 @@ export function lerp(a: number, b: number, t: number, p: number) {
+    switch (randNum) {
+      case (0): {
+        layer = new etro.layer.Image({
+          startTime: start,
+          duration: section.scriptDuration,
+          source: await resizeImage(section.scriptMedia.url, WIDTH, HEIGHT),
+          destX: (_element: etro.EtroObject, time: number) => {
+            return lerp(0, -WIDTH / 10, time, section.scriptDuration!);
+          }, // default: 0
+          destY: (_element: etro.EtroObject, time: number) => {
+            return lerp(0, -HEIGHT / 10, time, section.scriptDuration!);
+          }, // default: 0
+          destWidth: (_element: etro.EtroObject, time: number) => {
+            return lerp(WIDTH, WIDTH * 1.2, time, section.scriptDuration!);
+          }, // default: null (full width)
+          destHeight: (_element: etro.EtroObject, time: number) => {
+            return lerp(HEIGHT, HEIGHT * 1.2, time, section.scriptDuration!);
+          },
+          x: 0, // default: 0
+          y: 0, // default: 0
+          sourceWidth: WIDTH,
+          sourceHeight: HEIGHT,
+          opacity: 1, // default: 1
+        });
+        break;
       }
-      console.log("adding layer", layer);
-      start += section.scriptDuration;
-      movie.layers.push(layer!);
-    });
+      case (1): {
+        layer = new etro.layer.Image({
+          startTime: start,
+          duration: section.scriptDuration,
+          source: await resizeImage(section.scriptMedia.url, WIDTH, HEIGHT),
+          destX: (_element: etro.EtroObject, time: number) => {
+            return lerp(-WIDTH / 10, 0, time, section.scriptDuration!);
+          }, // default: 0
+          destY: (_element: etro.EtroObject, time: number) => {
+            return lerp(-HEIGHT / 10, 0, time, section.scriptDuration!);
+          }, // default: 0
+          destWidth: (_element: etro.EtroObject, time: number) => {
+            return lerp(WIDTH * 1.2, WIDTH, time, section.scriptDuration!);
+          }, // default: null (full width)
+          destHeight: (_element: etro.EtroObject, time: number) => {
+            return lerp(HEIGHT * 1.2, HEIGHT, time, section.scriptDuration!);
+          },
+          x: 0, // default: 0
+          y: 0, // default: 0
+          sourceWidth: WIDTH,
+          sourceHeight: HEIGHT,
+          opacity: 1, // default: 1
+        });
+        break;
+      }
+      default: {
+        layer = new etro.layer.Image({
+          startTime: start,
+          duration: section.scriptDuration,
+          source: await resizeImage(section.scriptMedia.url, WIDTH, HEIGHT),
+          destX: (_element: etro.EtroObject, time: number) => {
+            return lerp(0, -WIDTH / 5, time, section.scriptDuration!);
+          }, // default: 0
+          destY: 0, // default: 0
+          destWidth: WIDTH * 1.2, // default: null (full width)
+          destHeight: HEIGHT  *1.2,
+          x: 0, // default: 0
+          y: 0, // default: 0
+          sourceWidth: WIDTH,

```

Comment:
I think it would be a lot easier to just omit setting source width and source height rather than carrying around their widths everywhere